[2025-05-03T15:50:00.317+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-03T15:50:00.358+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews.run_dbt_models manual__2025-05-03T15:43:14.960585+00:00 [queued]>
[2025-05-03T15:50:00.383+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews.run_dbt_models manual__2025-05-03T15:43:14.960585+00:00 [queued]>
[2025-05-03T15:50:00.402+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-03T15:50:00.433+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): run_dbt_models> on 2025-05-03 15:43:14.960585+00:00
[2025-05-03T15:50:00.464+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=6131) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-03T15:50:00.470+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews', 'run_dbt_models', 'manual__2025-05-03T15:43:14.960585+00:00', '--job-id', '352', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmphal8nfrm']
[2025-05-03T15:50:00.472+0000] {standard_task_runner.py:72} INFO - Started process 6146 to run task
[2025-05-03T15:50:00.473+0000] {standard_task_runner.py:105} INFO - Job 352: Subtask run_dbt_models
[2025-05-03T15:50:00.573+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews.run_dbt_models manual__2025-05-03T15:43:14.960585+00:00 [running]> on host 8280c28b7f0f
[2025-05-03T15:50:00.761+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews' AIRFLOW_CTX_TASK_ID='run_dbt_models' AIRFLOW_CTX_EXECUTION_DATE='2025-05-03T15:43:14.960585+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-03T15:43:14.960585+00:00'
[2025-05-03T15:50:00.765+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-03T15:50:00.766+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-03T15:50:00.768+0000] {logging_mixin.py:190} INFO - Current task name:run_dbt_models state:running start_date:2025-05-03 15:50:00.359972+00:00
[2025-05-03T15:50:00.769+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews and current dag run status:running
[2025-05-03T15:50:00.770+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-03T15:50:00.773+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-05-03T15:50:00.776+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt_project/bank_reviews_dbt && dbt run --profiles-dir .']
[2025-05-03T15:50:00.807+0000] {subprocess.py:99} INFO - Output:
[2025-05-03T15:50:06.059+0000] {subprocess.py:106} INFO - [0m15:50:06  Running with dbt=1.9.4
[2025-05-03T15:50:06.711+0000] {subprocess.py:106} INFO - [0m15:50:06  Registered adapter: postgres=1.9.0
[2025-05-03T15:50:07.923+0000] {subprocess.py:106} INFO - [0m15:50:07  Found 5 models, 12 data tests, 433 macros
[2025-05-03T15:50:07.932+0000] {subprocess.py:106} INFO - [0m15:50:07
[2025-05-03T15:50:07.934+0000] {subprocess.py:106} INFO - [0m15:50:07  Concurrency: 1 threads (target='dev')
[2025-05-03T15:50:07.936+0000] {subprocess.py:106} INFO - [0m15:50:07
[2025-05-03T15:50:08.161+0000] {subprocess.py:106} INFO - [0m15:50:08  1 of 5 START sql table model review.dim_bank ................................... [RUN]
[2025-05-03T15:50:08.494+0000] {subprocess.py:106} INFO - [0m15:50:08  1 of 5 OK created sql table model review.dim_bank .............................. [[32mSELECT 20[0m in 0.33s]
[2025-05-03T15:50:08.504+0000] {subprocess.py:106} INFO - [0m15:50:08  2 of 5 START sql table model review.dim_branch ................................. [RUN]
[2025-05-03T15:50:08.657+0000] {subprocess.py:106} INFO - [0m15:50:08  2 of 5 OK created sql table model review.dim_branch ............................ [[32mSELECT 20[0m in 0.15s]
[2025-05-03T15:50:08.665+0000] {subprocess.py:106} INFO - [0m15:50:08  3 of 5 START sql table model review.dim_location ............................... [RUN]
[2025-05-03T15:50:08.821+0000] {subprocess.py:106} INFO - [0m15:50:08  3 of 5 OK created sql table model review.dim_location .......................... [[32mSELECT 20[0m in 0.15s]
[2025-05-03T15:50:08.826+0000] {subprocess.py:106} INFO - [0m15:50:08  4 of 5 START sql table model review.dim_sentiment .............................. [RUN]
[2025-05-03T15:50:08.971+0000] {subprocess.py:106} INFO - [0m15:50:08  4 of 5 OK created sql table model review.dim_sentiment ......................... [[32mSELECT 1[0m in 0.14s]
[2025-05-03T15:50:08.976+0000] {subprocess.py:106} INFO - [0m15:50:08  5 of 5 START sql table model review.fact_reviews ............................... [RUN]
[2025-05-03T15:50:09.182+0000] {subprocess.py:106} INFO - [0m15:50:09  5 of 5 OK created sql table model review.fact_reviews .......................... [[32mSELECT 196[0m in 0.20s]
[2025-05-03T15:50:09.215+0000] {subprocess.py:106} INFO - [0m15:50:09
[2025-05-03T15:50:09.216+0000] {subprocess.py:106} INFO - [0m15:50:09  Finished running 5 table models in 0 hours 0 minutes and 1.28 seconds (1.28s).
[2025-05-03T15:50:09.345+0000] {subprocess.py:106} INFO - [0m15:50:09
[2025-05-03T15:50:09.347+0000] {subprocess.py:106} INFO - [0m15:50:09  [32mCompleted successfully[0m
[2025-05-03T15:50:09.349+0000] {subprocess.py:106} INFO - [0m15:50:09
[2025-05-03T15:50:09.351+0000] {subprocess.py:106} INFO - [0m15:50:09  Done. PASS=5 WARN=0 ERROR=0 SKIP=0 TOTAL=5
[2025-05-03T15:50:12.036+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-05-03T15:50:12.087+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-03T15:50:12.089+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews, task_id=run_dbt_models, run_id=manual__2025-05-03T15:43:14.960585+00:00, execution_date=20250503T154314, start_date=20250503T155000, end_date=20250503T155012
[2025-05-03T15:50:12.146+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-03T15:50:12.148+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-03T15:50:12.149+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews queued_at:2025-05-03 15:43:15.053289+00:00
[2025-05-03T15:50:12.150+0000] {logging_mixin.py:190} INFO - Task hostname:8280c28b7f0f operator:BashOperator
[2025-05-03T15:50:12.197+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-03T15:50:12.246+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-03T15:50:12.252+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
